src/NosCore.GameObject/ComponentEntities/Extensions/CharacterEntityExtension.cs

```diff
@@ -247,7 +247,7 @@ public static InPacket GenerateIn(this ICharacterEntity visualEntity, string pre
                 Direction = visualEntity.Direction,
                 InCharacterSubPacket = new InCharacterSubPacket
                 {
-                    Authority = (byte)visualEntity.Authority, //todo change chickenapi to short
+                    Authority = visualEntity.Authority > AuthorityType.Administrator ? (byte)2 : (byte)visualEntity.Authority, //todo change chickenapi to short
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4515499)

Reply:
Instead of casting '2' to a byte, better cast AuthorityType.GameMaster to a byte. Magic Nubers are not really wanted though

(Even though a new enum would be better, as it's only for visibility purposes, this shouldn't be an AuthorityType)